app/src/main/res/xml-eu/ext_kbd_bottom_row_ng.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you think that the changes to the bottom rows are going to be problematic in the future I can remove them from the pull request. I don't want to cause you any inconvenience.
The hyphen and colon symbols are more used in Basque than the single quote and the semicolon, specially the hyphen, as it's used to add suffixes and to create compound words. We thought that it's more useful for Basque speakers that way.